core/src/test/java/co/cask/wrangler/steps/transformation/ExpressionTest.java

```diff
@@ -169,5 +181,36 @@ public void testMethodCalls() throws Exception {
     records = PipelineTest.execute(directives, records);
     Assert.assertTrue(records.size() == 0);
   }
+
+  @Test
+  public void testGeoFence() throws Exception {
+    Gson gson = new GsonBuilder()
+            .registerTypeAdapterFactory(new GeometryAdapterFactory())
+            .create();
+
+    List<SinglePosition> singlePositionList = new ArrayList<>();
+    singlePositionList.add(new SinglePosition(Coordinates.of(-466.14990234375, 43.8503744993026)));
+    singlePositionList.add(new SinglePosition(Coordinates.of(-464.1064453125, 42.76314586689492)));
+    singlePositionList.add(new SinglePosition(Coordinates.of(-462.41455078125, 43.628123412124616)));
+    singlePositionList.add(new SinglePosition(Coordinates.of(-466.14990234375, 43.8503744993026)));
+
+    AreaPositions positions = new AreaPositions(ImmutableList.of(new LinearPositions(singlePositionList)));
+    FeatureCollection fences = new FeatureCollection(ImmutableList.of(Feature.of(new Polygon(positions))));
+
+    Coordinates location = Coordinates.of(-462.49145507812494, 43.46089378008257);
+
+    String[] directives = new String[] {
```

Comment:
Every function in the above file is not in here please add them all or only keep the once that are needed. Don't add a bit of code if it's not going to be used.